apps/backend/src/user/user.service.ts

```diff
@@ -15,7 +17,7 @@ export class UserService {
    * TODO : passwords are stored clearly in database -> hash them !
    */
   async createUser(userDto: UserDto): Promise<number> {
-    const user = new UserEntity(userDto.username, userDto.password, userDto.role, userDto.active);
+    const user = new UserEntity(userDto.username, hashString(userDto.password), userDto.role, userDto.active);
```

Comment:
Beware, here you're not using your `hashString` function but the one imported from `'nx/src/lock-file/utils/hashing'` and it's probably not what you want !

Reply:
> see how to call static methods https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static